docs/easybuild-v5/run_shell_cmd.md

```diff
@@ -8,20 +8,82 @@ This function replaces both the `run_cmd` and `run_cmd_qa` functions, which will
 
 ## Motivation
 
-...
+Over the years `run_cmd` and `run_cmd_qa` accumulated a lot of arguments which sometimes have misleading names.
+Moreover they have two different kinds of return values (depending on whether called with `simple=True` or `simple=False`).
+So a new command `run_shell_cmd` is introduced to replace both, using more direct and natural arguments, and as a bonus,
+better error reporting. In line with `--trace` being set by default, `run_shell_cmd` is now also more verbose by default.
 
 ## High-level overview
 
-...
+First of all `run_shell_cmd` has been designed so the defaults can be used for most situations.
+In that case the main thing to watch out for is the return code, which changed from a tuple `(output, exit_code)`
+to a named tuple with three fields:
```

Comment:
There are way more than 3 fields currently:
```
RunShellCmdResult = namedtuple('RunShellCmdResult', ('cmd', 'exit_code', 'output', 'stderr', 'work_dir',
                                                     'out_file', 'err_file'))
```

Ongoing PRs may add more (cfr. https://github.com/easybuilders/easybuild-framework/pull/4444).

So for now, I would change this to:

```suggestion
to a named tuple with multiple fields, the most prominent ones being: